### PR TITLE
Back-port #48193 to 2017.7

### PR DESCRIPTION
**Pull request by rallytime**
Back-port #48193 to 2017.7

Refs #42659

### CODE REVIEW

**Review by gtmanfred — APPROVED** [review]

File: salt/netapi/rest_tornado/saltnado.py [hunk 1]
```diff
@@ -937,88 +939,91 @@ def _disbatch_local(self, chunk):
...
                 except Exception:
                     pass
             raise tornado.gen.Return('No minions matched the target. No command was sent, no jid was assigned.')
 
+        # Map of minion_id -> returned for all minions we think we need to wait on
+        minions = {m: False for m in pub_data['minions']}
+
+        # minimum time required for return to complete. By default no waiting, if
+        # we are a syndic then we must wait syndic_wait at a minimum
+        min_wait_time = Future().set_result(True)
```
Comment (DmitryKuzmenko) [inline salt/netapi/rest_tornado/saltnado.py]: This shall be:
```python
min_wait_time = Future()
min_wait_time.set_result(True)
```

The same is in the original PR.

Comment (DmitryKuzmenko) [reply salt/netapi/rest_tornado/saltnado.py]: It's the reason of the tests failures.

**Review by DmitryKuzmenko — CHANGES_REQUESTED** [review]
Sorry my fault. I've missed this in the original PR. =(

File: salt/netapi/rest_tornado/saltnado.py (hunk 1 above)
Comment (DmitryKuzmenko) [reply salt/netapi/rest_tornado/saltnado.py]: @jacksontj could you please confirm my point?

**Review by DmitryKuzmenko — APPROVED** [review]

**Pull request merged**